registration_ref/app.py

```diff
@@ -46,8 +46,12 @@ def create_in_foundries(client_cert: str, api_token: str, name: Optional[str] =
         "OSF-TOKEN": api_token,
     }
     for x in (0.1, 0.2, 1, 0):
+        log.info("Initiating request")
```

Comment:
Not sure how useful is this log message, as it gives no context except a time.

Reply:
Removed